src/main/scala/org/broadinstitute/dsde/firecloud/model/Workspace.scala

```diff
@@ -28,10 +28,12 @@ case class EntityId(entityType: String, entityName: String)
 
 case class BagitImportRequest(bagitURL: String, format: String)
 
-// the request payload sent by users to Orchestration for async PFB imports
-case class AsyncImportRequest(url: Option[String])
+// legacy class specific to PFB import; prefer AsyncImportRequest instead
+case class PFBImportRequest(url: String)
+// the request payload sent by users to Orchestration for async PFB and TDR snapshot imports
+case class AsyncImportRequest(url: String, filetype: String)
```

Comment:
changing `AsyncImportRequest.url` from `Option[String]` to `String` caused lots of other lines of code to change, but I think it's a good change